config/gcpmanager/gcp-manager.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: gcp-manager
+  namespace: default
```

Comment:
The `gcp-manager` is a part of the Substratus control plane and should live in the `substratus` namespace.

Reply:
I hadn't considered that but spot on